definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -10320,5 +10320,16 @@
     "PageID": "ReturnToPreviousState1",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "20/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
```

Comment:
shouldn't there also be an amend entry?

Reply:
I don't know. I just moved all from nonprod file to the core.

Apparently amend has not been feature toggled and is already in the core file.

Reply:
just checked, it's in the resp-journey nonprod config, I think it needs to be toggled on as part of share a case rather than resp journey?

Reply:
displaying this value on tab is also in resp-journey nonprod so might want to toggle both on